lib/svtplay_dl/utils/check_ffmpeg.py

```diff
@@ -0,0 +1,63 @@
+import os, sys
+from subprocess import check_output
+from pkg_resources import parse_version
+from re import findall, sub, split
+from requests import get
+from svtplay_dl.utils import which
+from svtplay_dl import log
+
+def is_old():
+    if not which('ffmpeg'):
+        return None
+    version_check = check_output([which('ffmpeg'),'-version']).split('\n')[0].split(' ')[:3]
+    version_check = [v for v in version_check if findall('\d+', v)][0]
+    if 'N' in version_check[0]:
+        k = 1 if version_check.endswith('static') else 0
+        version_check = version_check.split('-')[::-1][k][1:]
+
+        r = get('https://api.github.com/repos/FFmpeg/FFmpeg/commits/' + version_check)
+        print ' '.join(split('T|Z', r.json()['commit']['committer']['date'])[:2])
+        if ' '.join(split('T|Z', r.json()['commit']['committer']['date'])[:2]) < "2012-09-28 01:02:28":
+            return True
+    else:
+        if not version_check.endswith('.git'):
+            version_check = version_check.split('-')[0]
+        else:
+            version_check = version_check.replace('.git','')
+        if parse_version(version_check) <= parse_version('1.0'):
+            return True
+
+def get_ffmpeg():
+    from platform import system
```

Comment:
oh haha, why is that? i think i've read somewhere that imports are being done from start anyway, am i right?